From: David Milburn <dmilburn@redhat.com>
Date: Thu, 18 Nov 2010 20:40:52 -0500
Subject: [misc] x86/PCI irq and pci_ids for Intel Patsburg Devices
Message-id: <1290112856-1796-6-git-send-email-dmilburn@redhat.com>
Patchwork-id: 29507
O-Subject: [RHEL5.6 PATCH 5/9] BZ 570868 x86/PCI irq and pci_ids patch for Intel
	Patsburg DeviceIDs
Bugzilla: 570868
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Michal Schmidt <mschmidt@redhat.com>

commit 25143fd1270d28782ae0620aa86ef5f8c14030fd
Author: Seth Heasley <seth.heasley@intel.com>
Date:   Fri Sep 10 16:36:39 2010 -0700

    x86/PCI: irq and pci_ids patch for Intel Patsburg DeviceIDs

    This patch adds the LPC Controller DeviceIDs for the Intel Patsburg PCH.

BZ 570868

diff --git a/arch/i386/pci/irq.c b/arch/i386/pci/irq.c
index e6ee7a1..b0cd69d 100644
--- a/arch/i386/pci/irq.c
+++ b/arch/i386/pci/irq.c
@@ -555,6 +555,7 @@ static __init int intel_router_probe(struct irq_router *r, struct pci_dev *route
 		case PCI_DEVICE_ID_INTEL_ICH10_1:
 		case PCI_DEVICE_ID_INTEL_ICH10_2:
 		case PCI_DEVICE_ID_INTEL_ICH10_3:
+		case PCI_DEVICE_ID_INTEL_PBG_LPC:
 			r->name = "PIIX/ICH";
 			r->get = pirq_piix_get;
 			r->set = pirq_piix_set;
diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index d5bed5e..4ffe59e 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2220,6 +2220,7 @@
 #define PCI_DEVICE_ID_INTEL_CPT_SMBUS	0x1c22
 #define PCI_DEVICE_ID_INTEL_CPT_LPC_MIN	0x1c41
 #define PCI_DEVICE_ID_INTEL_CPT_LPC_MAX	0x1c5f
+#define PCI_DEVICE_ID_INTEL_PBG_LPC	0x1d40
 #define PCI_DEVICE_ID_INTEL_82801AA_0	0x2410
 #define PCI_DEVICE_ID_INTEL_82801AA_1	0x2411
 #define PCI_DEVICE_ID_INTEL_82801AA_3	0x2413
